### ISSUE TITLE
Checkpoint files are not written to /data

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#374]
Checkpoints are written under the home dir. As home isn't large this causes it to run out of space....
```
…
InternalError (see above for traceback): Error writing (tmp) checkpoint file: /home/doyouunderstand/.local/share/deepspeech_wer/checkpoint/205aed8/model.ckpt-6.tempstate9784191321997661396: Resource exhausted: /home/doyouunderstand/.local/share/deepspeech_wer/checkpoint/205aed8/model.ckpt-6.tempstate9784191321997661396
	 [[Node: save/save = SaveSlices[T=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save/Const_0, save/save/tensor_names, save/save/shapes_and_slices, BiRNN/BW/BasicLSTMCell/Linear/Bias/_1085, BiRNN/BW/BasicLSTMCell/Linear/Bias/Adam/_1087, BiRNN/BW/BasicLSTMCell/Linear/Bias/Adam_1/_1089, BiRNN/BW/BasicLSTMCell/Linear/Matrix/_1091, BiRNN/BW/BasicLSTMCell/Linear/Matrix/Adam/_1093, BiRNN/BW/BasicLSTMCell/Linear/Matrix/Adam_1/_1095, BiRNN/FW/BasicLSTMCell/Linear/Bias/_1097, BiRNN/FW/BasicLSTMCell/Linear/Bias/Adam/_1099, BiRNN/FW/BasicLSTMCell/Linear/Bias/Adam_1/_1101, BiRNN/FW/BasicLSTMCell/Linear/Matrix/_1103, BiRNN/FW/BasicLSTMCell/Linear/Matrix/Adam/_1105, BiRNN/FW/BasicLSTMCell/Linear/Matrix/Adam_1/_1107, b1, b1/Adam, b1/Adam_1, b2, b2/Adam, b2/Adam_1, b3, b3/Adam, b3/Adam_1, b5, b5/Adam, b5/Adam_1, b6, b6/Adam, b6/Adam_1, beta1_power, beta2_power, h1, h1/Adam, h1/Adam_1, h2, h2/Adam, h2/Adam_1, h3, h3/Adam, h3/Adam_1, h5, h5/Adam, h5/Adam_1, h6, h6/Adam, h6/Adam_1)]]

Traceback (most recent call last):
  File "<stdin>", line 600, in <module>
  File "<stdin>", line 577, in exec_main
  File "<stdin>", line 497, in exec_wer_run
  File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['./bin/run-wer-automation.sh']' returned non-zero exit status 1
```
Also
```
doyouunderstand@ubuntu:~$ df -h
Filesystem      Size  Used Avail Use% Mounted on
udev             32G  4.0K   32G   1% /dev
tmpfs           6.3G  1.9M  6.3G   1% /run
/dev/md2        211G  200G  165M 100% /
none            4.0K     0  4.0K   0% /sys/fs/cgroup
none            5.0M     0  5.0M   0% /run/lock
none             32G  144K   32G   1% /run/shm
none            100M   28K  100M   1% /run/user
/dev/md0        181M   83M   85M  50% /boot
/dev/md3        7.3T  659G  6.3T  10% /data
```

**Comment by kdavis-mozilla** [comment]
@lissyx Can you take this and make sure the cron scripts handle this correctly.

**Comment by kdavis-mozilla** [comment]
@lissyx I guess the most important thing is to make the env variables of the `run-wer-automation.sh` save the checkpoints to `/data` and to make the other cron scripts work with this change.

**Comment by lissyx** [comment]
Sure, should be fine for this, but I guess I'll have to ensure what happens before checkpoint is being written ; we probably don't want to put pressure on /data the same way.

Maybe we should also set TMP=/data/tmp in the cron and have a /data/tmp to take care of this? The rationale being that /tmp is not big either :)

**Comment by kdavis-mozilla** [comment]
@lissyx Setting `TMP` sounds like a good plan to me.

**Comment by lissyx** [comment]
* PR has been merged
 * data have been copied from /home/doyouunderstand/.local/share/deepspeech_wer/ to /data/deepspeech_wer/
 * Crontab entry has been augmented with ds_dataroot="/data/"

**Comment by lissyx** [comment]
PR for merging the changes on the WER branch is at: https://github.com/mozilla/DeepSpeech/pull/378

**Comment by lissyx** [comment]
This will be fixed when issue #340 gets merged to WER branch.

**Comment by lissyx** [comment]
This landed in wer-tracking branch: https://github.com/mozilla/DeepSpeech/commit/5e377de6772ab3ad293cb110f24c5dd5f3e2a514
